native/src/main/java/io/ballerina/stdlib/protobuf/nativeimpl/AnyTypeCreator.java

```diff
@@ -63,50 +82,127 @@ public static Object unpack(BMap<BString, Object> value, BTypedesc targetType) {
+                        Descriptors.Descriptor descriptor = com.google.protobuf.WrappersProto.getDescriptor()
+                                .findMessageTypeByName(getMessageNameFromTypeUrl(typeUrl));
+                        DeserializeHandler deserializeHandler = new DeserializeHandler(descriptor,
+                                value.getStringValue(StringUtils.fromString("value")).getValue(),
+                                targetType.getDescribingType(), null);
+                        deserializeHandler.deserialize();
+                        return deserializeHandler.getBMessage();
+                    case EMPTY_TYPE_URL:
+                        return null;
+                    case TIMESTAMP_TYPE_URL:
+                        Descriptors.Descriptor timestampDescriptor = com.google.protobuf.Timestamp.getDescriptor();
+                        DeserializeHandler timestampDeserializeHandler = new DeserializeHandler(timestampDescriptor,
+                                value.getStringValue(StringUtils.fromString("value")).getValue(),
+                                targetType.getDescribingType(), targetType.getDescribingType());
+                        timestampDeserializeHandler.deserialize();
+                        BArray utcTime = (BArray) timestampDeserializeHandler.getBMessage();
+                        utcTime.freezeDirect();
+                        return utcTime;
+                    case DURATION_TYPE_URL:
+                        Descriptors.Descriptor durationDescriptor = com.google.protobuf.Duration.getDescriptor();
+                        DeserializeHandler durationDeserializeHandler = new DeserializeHandler(durationDescriptor,
+                                value.getStringValue(StringUtils.fromString("value")).getValue(),
+                                targetType.getDescribingType(), targetType.getDescribingType());
+                        durationDeserializeHandler.deserialize();
+                        return durationDeserializeHandler.getBMessage();
+                    case STRUCT_TYPE_URL:
+                        Descriptors.Descriptor structDescriptor = com.google.protobuf.Struct.getDescriptor();
+                        DeserializeHandler structDeserializeHandler = new DeserializeHandler(structDescriptor,
+                                value.getStringValue(StringUtils.fromString("value")).getValue(),
+                                targetType.getDescribingType(), targetType.getDescribingType());
+                        structDeserializeHandler.deserialize();
+                        return structDeserializeHandler.getBMessage();
+                    case ANY_TYPE_URL:
+                        Descriptors.Descriptor anyDescriptor = com.google.protobuf.Any.getDescriptor();
+                        DeserializeHandler anyDeserializeHandler = new DeserializeHandler(anyDescriptor,
+                                value.getStringValue(StringUtils.fromString("value")).getValue(),
```

Comment:
we can use a constant here also
StringUtils.fromString("value")

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-protobuf/pull/176/commits/0b4e304a70313de3a78afb5bc4ffe5ce99f8cd6b